src/reducers/scenes/SendConfirmationReducer.js

```diff
@@ -245,6 +247,17 @@ export const transaction = (state: EdgeTransaction | null = null, action: Action
   }
 }
 
+export const toggleFiatOnTop = (state: boolean | null = null, action: Action) => {
+  switch (action.type) {
+    case 'UI/SEND_CONFIMATION/TOGGLE_FIAT_ON_TOP': {
+      return action.data
+    }
+
+    default:
+      return state
```

Comment:
Once this gets set, it never gets unset, which means the flip input would be stuck flipped forever, I think.

Reply:
Yeah I guess so, fixed to default state null